tortoise/tortoise.go

```diff
@@ -54,7 +53,7 @@ func newTurtle(
 		logger:  logger,
 		cdb:     cdb,
 		beacons: beacons,
-		updater: updater,
+		updated: make([]types.BlockContextualValidity, 0),
```

Comment:
updated can be nil

Reply:
also in general make with 0 doesn't make sense, it is unfortunate that this syntax is allowed